<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivePlatformClass</class>
 <widget class="QMainWindow" name="LivePlatformClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>556</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LivePlatform</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LiveLoginDialog.qrc">
    <normaloff>:/LiveLoginDialog/res/live.ico</normaloff>:/LiveLoginDialog/res/live.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">/* font */
*
{
	font-family: &quot;Arial&quot;, &quot;Segoe UI&quot;, &quot;黑体&quot;, &quot;等线&quot;;
	font-size: 14px;
	font-weight: 400;
	line-height: 20px;
}
/* background */
QWidget#centralWidget
{
	background: rgba(36,41,46,0.9);
	background-position: top;
	background-image: url(:/LiveLoginDialog/res/login-background-800x240.png);
	background-repeat: no-repeat;
}

QLabel#streamServerLabel,#cameraLabel,#microphoneLabel, QComboBox
{
	color:white;
	border: 2px solid rgba(27,128,255,0.5);
	background: rgba(45,64,72,0.8);
}

QLabel#applicationIconLabel
{
	border-image: url(:/LiveLoginDialog/res/pilipili-title-210x50.png);
}
/* avatar */
QLabel#avatarLabel
{
	border-image: url(:/LiveLoginDialog/res/avatar-mask.png);
	border-radius:60px;
	min-width:  120px;
    max-width:  120px;
    min-height: 120px;
    max-height: 120px;
    border-width: 0 0 0 0;
}

/* button */
.QPushButton, .QCheckBox, .QToolButton
{
	color: white;
	background-color: transparent;
}
/*RadioButton CheckButton*/
QRadioButton::indicator::unchecked {
	image: url(:/LiveLoginDialog/res/radiobutton-unchecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/LiveLoginDialog/res/radiobutton-checked.png);
}
QCheckBox::indicator { 
	width: 15px;
	height: 30px;
}
QCheckBox::indicator::unchecked {	
	image: url(:/LiveLoginDialog/res/checkbox_unchecked.png);
}
QCheckBox::indicator::checked { 
	image: url(:/LiveLoginDialog/res/checkbox_checked.png);
}

.QCheckBox::indicator { 
	width: 15px;
	height: 15px;
}
.QCheckBox::indicator::unchecked {	
	image: url(:/LiveLoginDialog/res/radiobutton-unchecked.png);
}
.QCheckBox::indicator::checked { 
	image: url(:/LiveLoginDialog/res/radiobutton-checked.png);
}
.QPushButton:hover, .QCheckBox:hover
{
	color: yellow;
}
.QPushButton:pressed, .QCheckBox:pressed
{
	color: #FFCC66;
}
.QPushButton:focus, .QCheckBox:focus
{
	outline: none
}
/* start stop button */
QPushButton#startPushButton {
  background: #0078d7;
  border: 2px solid rgba(0,0,0,0.4)
}
QPushButton#startPushButton:hover {
  background: #1CA8FA;
}
QPushButton#startPushButton:pressed {
  background: #2A67FF;
}
QPushButton#startPushButton:disabled {
  background: #666;
}
QPushButton#stopPushButton {
  background: #d77800;
  border: 2px solid rgba(0,0,0,0.4)
}
QPushButton#stopPushButton:hover {
  background: #FAA81C;
}
QPushButton#stopPushButton:pressed {
  background: #FF672A;
}
QPushButton#stopPushButton:disabled {
  background: #666;
}

/* account password */
QLineEdit
{
	border: 2px solid rgba(27,128,255,0.5);
	background: rgba(45,64,72,0.8);
	color: white;
    selection-background-color:green;
}
QLineEdit:focus
{
	background: rgba(80,100,110,0.8);
}
/* tooltip */
QToolTip
{
	color: yellow;
	border: 2px solid #333;
	background-color: black;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titlebarWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="applicationIconLabel">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>50</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="titlebarHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>303</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="titleMenuVerticalLayoutWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="titleMenuVerticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="titleMenuWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>32</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="titleMenuHorizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="minimizeToolButton">
               <property name="minimumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>最小化</string>
               </property>
               <property name="icon">
                <iconset resource="LiveLoginDialog.qrc">
                 <normaloff>:/LiveLoginDialog/res/minus-square.png</normaloff>:/LiveLoginDialog/res/minus-square.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="closeToolButton">
               <property name="minimumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>关闭</string>
               </property>
               <property name="icon">
                <iconset resource="LiveLoginDialog.qrc">
                 <normaloff>:/LiveLoginDialog/res/close-square.png</normaloff>:/LiveLoginDialog/res/close-square.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="customWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="controlWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>691</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>91</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="microphoneLabel">
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>Microphone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="microphoneComboBox">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="streamServerLabel">
              <property name="minimumSize">
               <size>
                <width>96</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="streamServerLineEdit">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>rtmp.thatboy.info:27986</string>
              </property>
              <property name="placeholderText">
               <string>Server here.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="cameraLabel">
              <property name="minimumSize">
               <size>
                <width>108</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>108</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>Camera</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cameraComboBox">
              <property name="minimumSize">
               <size>
                <width>240</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>320</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="startPushButton">
              <property name="minimumSize">
               <size>
                <width>96</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>开播</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopPushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>96</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>停播</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="sizeComboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="cameraViewWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>300</height>
          </size>
         </property>
         <layout class="QGridLayout" name="cameraViewLayout"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LivePlatform.qrc"/>
  <include location="LiveLoginDialog.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>false</bool>
  </property>
 </designerdata>
</ui>
